### PR TITLE
Network, Channels and Community are children of timeline

### PR DESCRIPTION
**Pull request by annando**
There is now a central model class for all timelines. It contains all common functions. Also for authenticated users the community feeds now moved to the channels to save space.

There are several preparations to be able to call channels and community streams from the network page.

This is some work in progress, but should be mergeable in any point in time.

### CODE REVIEW

File: src/Content/Nav.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ class Nav
 		'directory' => null,
 		'settings'  => null,
 		'contacts'  => null,
-		'delegation'=> null,
+		'delegation' => null,
```
Comment (MrPetovan) [inline src/Content/Nav.php]: If you're changing this, please align all the arrows in this array declaration.

File: src/Module/Conversation/Timeline.php [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+	/** @var string */
+	protected static $accountTypeString;
+	/** @var int */
+	protected static $accountType;
+	/** @var int */
+	protected static $item_id;
+	/** @var int */
+	protected static $itemsPerPage;
+	/** @var bool */
+	protected static $no_sharer;
```
Comment (MrPetovan) [inline src/Module/Conversation/Timeline.php]: If these are only called in object context, they don't need to be static properties anymore.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Module/Conversation/Timeline.php (hunk 2 above)
Comment (annando) [reply src/Module/Conversation/Timeline.php]: It's done for community and channels, but not for the network. That class needs some work.

Comment (annando) [reply src/Module/Conversation/Timeline.php]: Done

**Comment by MrPetovan** [comment]
The translation file needs to be regenerated.

**Comment by annando** [comment]
@MrPetovan fixed.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**